core/trino-main/src/main/java/io/trino/server/ServerMainModule.java

```diff
@@ -205,6 +206,11 @@ protected void setup(Binder binder)
         }
 
         binder.bind(StartupStatus.class).in(Scopes.SINGLETON);
+
+        configBinder(binder).bindConfigDefaults(HttpServerConfig.class, httpServerConfig -> {
+            httpServerConfig.setHttp2MaxConcurrentStreams(32 * 1024); // from the default 16K
```

Comment:
why not bump in airlift?

Reply:
I will move it to airlift later as a follow up